### PR TITLE
Ml

### PR DESCRIPTION
**Pull request by LelandCurtis**
- add ml fair_price service
- add predictive price trendline that reacts to each model type

### CODE REVIEW

**Comment by LelandCurtis** [comment]
Merging with main caused many issues that appear at first glance to be actual errors caused by an accidental merge to main. For example, there was a save_and_open_page. I x'd out the following tests so that the ml branch woudl work, but these need to be revisited:

**Comment by LelandCurtis** [comment]
there were issues with the main that I merged. It seemed like there could have been an accidental merge to main.

File: app/poros/listings.rb [hunk 1]
```diff
@@ -1,35 +0,0 @@
-class Listings
```
Comment (tjroeder) [inline app/poros/listings.rb:1]: Getting rid of the cruft. Nice.

**Review by croixk — APPROVED** [review]

**Review by Eldridge-Turambi — APPROVED** [review]
Thank you for doing this

File: app/poros/listings.rb (hunk 1 above)
Comment (LelandCurtis) [reply app/poros/listings.rb:1]: haha yea it was time

**Review by croixk — APPROVED** [review]

**Review by Eldridge-Turambi — APPROVED** [review]

**Review by kevingloss — APPROVED** [review]

**Pull request merged**